### PR TITLE
chore: code clean

### PR DESCRIPTION
**Pull request by hoho2017**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [x ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

-----
[View rendered components/table/demo/virtual-list.md](https://github.com/hoho2017/ant-design/blob/zhanghaoqiang/components/table/demo/virtual-list.md)

### CODE REVIEW

**Comment by hoho2017** [comment]
make code perfect

File: components/table/demo/virtual-list.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ function VirtualTable(props) {
         }}
         height={scroll.y}
         rowCount={rawData.length}
-        rowHeight={() => 54}
+        rowHeight={54}
```
Comment (yoyo837) [inline components/table/demo/virtual-list.md:77]: 这里好像改过 又撤销了

Comment (yoyo837) [reply components/table/demo/virtual-list.md:77]: #25740

Comment (hoho2017) [reply components/table/demo/virtual-list.md:77]: 为啥呀

Comment (yoyo837) [reply components/table/demo/virtual-list.md:77]: image

Comment (hoho2017) [reply components/table/demo/virtual-list.md:77]: you are right
image

**Pull request closed without merging**